docs/layers.md

```diff
@@ -511,6 +511,7 @@ _TIP: Some `landuse` features only exist as point features in OpenStreetMap. Fin
 * `dog_park`
 * `embankment` - A raised area of land, usually to carry a road or railway.
 * `enclosure`
+* `environmental_zone` - An area beloging to a protected environmental area, such as the (London Loz Emission Zone)[https://en.wikipedia.org/wiki/London_low_emission_zone].
```

Comment:
Maybe there's a better term for this like `restricted_driving_zone`?

Reply:
Not sure about that, as in general these zones are either `low_emission_zone` or `zero_emission_zone` [see here](https://en.wikipedia.org/wiki/Low-emission_zone).
That is why we opted for `environmental_zone` as more neutral as it is an area with environmental restrictions, but I get the point that perhaps it is not descriptive enough.
Should we just go with `low_emission_zone` as in [OSM](https://wiki.openstreetmap.org/wiki/Tag:boundary%3Dlow_emission_zone)?

Reply:
Yes, let's go with OSM's `low_emission_zone` terminology.

Reply:
Good, this is the term I used in my updated commit :)